examples/dbt_dagster_tutorial/README.md

```diff
@@ -0,0 +1,34 @@
+# dbt with software-defined assets tutorial example
+
+This example is meant to be used alongside the [Using dbt with Dagster software-defined assets tutorial](https://docs.dagster.io/integrations/dbt/using-dbt-with-dagster).
+
+The tutorial associated with this example demonstrates how to integrate dbt with Dagster using dbt's example [jaffle shop project](https://github.com/dbt-labs/jaffle_shop), the [dagster-dbt library](/\_apidocs/libraries/dagster-dbt), and a [DuckDB database](https://duckdb.org/).
+
+This example contains:
+
+- A blank template project (located in `tutorial_template`) that can be used to follow along with the tutorial
+- A finished project (`tutorial_finished`) that contains the completed project produced by the tutorial
+
+---
+
+## Getting started
+
+To download this example, run:
+
+```shell
+dagster project from-example --name my-dagster-project --example dbt_dagster_tutorial
```

Comment:
oh sorry - real one last comment: @jamiedemaria example code wise, can we rename`dbt_dagster_tutorial` to `tutorial_dbt_dagster`? 

asking because i could imagine us introducing more tutorials like this especially for integrations. as we write more, prefixing them with `tutorial_` will make those examples show up nicely in alphabetical order in https://github.com/dagster-io/dagster/tree/master/examples

this should be a quick grep. we should do it before merging because it'll be hard to change once it's released. let me know if you'd want me to make the changes.